packages/tools/pluggable-widgets-tools/src/typings-generator/__tests__/outputs/association.ts

```diff
@@ -19,7 +19,11 @@ export interface MyWidgetContainerProps {
 }
 
 export interface MyWidgetPreviewProps {
+    /**
+     * @deprecated since version 9.14.0. Please use class property instead.
```

Comment:
Did it change since Mendix 9.14 already or since Mendix 9.17?

Reply:
It's gonna be the case from 9.18 onwards, Sorry I guess I mentioned the tools version here instead. It should be SP version, am I right?

Reply:
We try to keep them in sync for major changes, but it is better to still refer to version of SP as tool version is a bit obscure to the user.